### ISSUE TITLE
All the tabs on the trace viewer page is not accessible with keyboard key: A11y_Playwright OSS releases_Keyboard

### ISSUE THREAD

**Issue by ganagdhar123** [microsoft/playwright.dev#1810]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags:
#A11yTCS;#Playwright OSS Releases;#BM_PlaywrightOSSReleases_External_Sep2025;#ChromiumEdge;#DesktopWeb;#FTP;#Win11;#A11ySev1;#A11yMAS;#WCAG2.1.1;
### Environmental Details:
App Name: Playwright OSS releases
#URL: https://playwright.dev/
OS & Browser Details:
Microsoft Windows 11 Enterprise
Version 26100.3323
Microsoft Edge
Version 131.0.2903.99 (Official build) (64-bit)
### Repro Steps:

1. Hit the URL: https://playwright.microsoft.com/workspaces/eastus_cdbdcaba-d6e1-47a0-80b2-e586bb4a7247 and sign in with workspace account.
2. "Home" page will appear. TAB to any "Test run" and hit ENTER key.
3. TAB to any test case and hit ENTER key. TAB to "View trace viewer" button and hit ENTER key.
4. TAB to "Actions" Metadata" Tab and tabs present beside "Pick locator" Tab.
5. Observe that All the tabs on the trace viewer section is accessible with keyboard key or not.

 ### Actual:
"Actions", "Metadata" and Tabs present beside the "Pick locator" tabs i.e, "Action", "before" and "After" Tabs on the trace viewer page is not accessible with keyboard "Tab and Arrow" keys.
Note: Trace viewer section is not accessible with keyboard.as we are able to drag and select but not accessible with keyboard
### Expected:
"Actions", "Metadata" and Tabs present beside the "Pick locator" tabs i.e, "Action", "before" and "After" Tabs on the trace viewer page should be accessible with keyboard "Tab and Arrow" keys.
### User Impact:
Keyboard only users will face difficulty in accessing the control and miss the information in it, if it is not accessible by keyboard.
### Attachment:

[2305990 Issues.zip](https://github.com/user-attachments/files/21313894/2305990.Issues.zip)
